_el/exercises.md

```diff
@@ -336,172 +204,52 @@ piece of code with the output it produces".
+
+Προκειμένου να δίνει τέτοιου είδους ανατροφοδότηση, το σύστημα των [[Buff2015](#CITE)] προσδιορίζει ποια μέθοδο ή μεθόδους του κώδικα του εκπαιδευόμενου οδηγούν στα αποτυχημένα τεστ, έτσι ώστε το σύστημα να μπορεί να συσχετίσει δοκιμές που απέτυχαν με συγκεκριμένα χαρακτηριστικά της υποβολής του εκπαιδευόμενου. Το σύστημα αποφασίζει εάν συγκεκριμένες νύξεις έχουν "κερδίσει" οι μαθητές, βλέποντας ένα έχουν πράγματι δοκιμάσει επαρκώς το σχετικό χαρακτηριστικό, ούτως ώστε οι εκπαιδευόμενοι να μην μπορούν να βασίζονται στις νύξεις αντί να κάνουν δοκιμές.
+
+Οι [[Keun2016a](#CITE),[Keun2016b](#CITE)] ταξινόμησαν τα μηνύματα που παράγουν 69 εργαλεία αυτόματης βαθμολόγησης. Διαπίστωσαν ότι αυτές συχνά δεν δίνουν ανατροφοδότηση στο πώς να διορθωθούν τα προβλήματα ώστε να γίνει το επόμενο βήμα. Βρήκαν επίσης ότι οι περισσότεροι εκπαιδευτικοί δεν μπορούν εύκολα να προσαρμόσουν περισσότερα από τα εργαλεία στις ανάγκες τους - όπως και πολλοί άλλα εργαλεία ροής εργασίας, τείνουν να επιβάλλουν τις μη ευσταθείς υποθέσεις των δημιουργών τους σχετικά με τον τρόπο λειτουργίας των εκπαιδευτικών οργανισμών. Οι συγγραφείς των άρθρων αυτών συνεχίζουν το έργο τους σε αυτό το χώρο και το λεπτομερές σύστημα ταξινόμησης τους είναι πολύ χρήσιμο για τον εντοπισμό των κατάλληλων εργαλείων αυτόματης βαθμολόγησης.
+
+Οι [[Srid2016](#CITE)] συζητούν στρατηγικές ανταλλαγής σχολίων με τους μαθητές κατά τη διάρκεια της αυτόματης βαθμολόγησής τους. Μια πρώτη στρατηγική είναι να δίνεται το αναμενόμενο αποτέλεσμα των δοκιμών -- αλλά στη συνέχεια οι μαθητές μπορούν να ενσωματώσουν αυτό το αποτέλεσμα στον κώδικά τους (γιατί ότι μπορεί να λειτουργήσει ως εύκολη λύση, θα γίνει). Μια εναλλακτική είναι η αναφορά των αποτελεσμάτων επιτυχίας / αποτυχίας για τον κώδικα των εκπαιδευομένων, αλλά να δίνονται τα πραγματικά αποτελέσματα των δοκιμνών (ως είσοδος / έξοδος του προγράμματος) μετά την καταλληκτική ημερομηνία υποβολής. Αυτό μπορεί να είναι απογοητευτικό, διότι τους λέει στους μαθητές ότι είναι λάθος, αλλά όχι το γιατί είναι.
```

Comment:
δοκιμνών--> δοκιμών

Reply:
Τhanks @pdelias ! Fixed in a new commit.